inc/render/class-woo-comparison-block.php

```diff
@@ -0,0 +1,143 @@
+<?php
+/**
+ * Woo_Comparison_Block
+ *
+ * @package ThemeIsle\GutenbergBlocks\Render
+ */
+
+namespace ThemeIsle\GutenbergBlocks\Render;
+
+use ThemeIsle\GutenbergBlocks\Base_Block;
+
+/**
+ * Class Woo_Comparison_Block
+ */
+class Woo_Comparison_Block extends Base_Block {
+
+	/**
+	 * Woo_Comparison_Block constructor.
+	 *
+	 * @since   1.7.0
+	 * @access  public
+	 */
+	public function __construct() {
+		parent::__construct();
```

Comment:
I didn't see any check about if the comparison table feature is enabled. I think we should check that. You can use the (`Options::is_module_activated`) method that in the following to check it.

https://github.com/Codeinwp/neve-pro-addon/blob/master/includes/modules/woocommerce_booster/comparison_table/options.php#L24-L31